esmvaltool/recipes/examples/recipe_check_obs.yml

```
@@ -65,11 +65,33 @@ diagnostics:
   CRU:
     description: CRU check
     variables:
-      tas:
-      pr:
-    additional_datasets:
-      - {dataset: CRU, project: OBS, mip: Amon, tier: 2,
-         type: reanaly, version: TS4.02, start_year: 1901, end_year: 2017}
+      tas:  # check older versions
+        mip: Amon
+        additional_datasets:
+        - {dataset: CRU, project: OBS, tier: 2, type: reanaly, 
+          version: TS4.02, start_year: 1901, end_year: 2017}
+        - {dataset: CRU, project: OBS6, tier: 2, type: reanaly, 
+          version: TS4.06, start_year: 1901, end_year: 2021}
```

```suggestion
          version: TS4.07, start_year: 1901, end_year: 2021}
```
since the new default with this pull request is 4.07, change it to this?